src/parse_file.py

```diff
@@ -281,40 +295,45 @@ def process_label(label: str, *, glabel: bool) -> None:
             elif ifdef_level == 0:
                 if line.startswith(".section"):
                     curr_section = line.split(" ")[1].split(",")[0]
-                    if curr_section == ".late_rodata":
+                    if curr_section in (".rdata", ".late_rodata"):
                         curr_section = ".rodata"
                 elif (
                     line.startswith(".rdata")
                     or line.startswith(".rodata")
+                    or line.startswith(".rdata")
```

Comment:
already checked two lines up

(`.section .rdata` is also invalid I believe, but it's fine to support)